### PR TITLE
feat: at exporter initialization, warn user if Phoenix is not running

### LINKED ISSUE #1487: detect whether Phoenix is running when initializing the HttpExporter for the tracer
When initializing the tracer with HttpExporter, the user should be informed if Phoenix is not running.

### PR DESCRIPTION
**Pull request by RogerHYang**
resolves #1487

### CODE REVIEW

**Review by axiomofjoy — APPROVED** [review]

File: src/phoenix/server/app.py [hunk 1]
```diff
@@ -157,6 +162,7 @@ def create_app(
             ]
         )
         + [
+            Route("/arize_phoenix_version", version),
```
Comment (mikeldking) [inline src/phoenix/server/app.py:165]:
```suggestion
            Route("/version", version),
```

File: src/phoenix/trace/exporter.py [hunk 2]
```diff
@@ -69,3 +89,12 @@ def _send(self, span: Span) -> None:
...
             logger.exception(e)
+
+    def _raise_if_phoenix_is_not_running(self) -> None:
+        try:
+            requests.get(f"{self._base_url}/arize_phoenix_version").raise_for_status()
+        except Exception:
+            raise Exception(
+                f"Arize Phoenix is not running on {self._base_url}. Launch Phoenix "
+                f"with `import phoenix as px; px.launch_app()`"
+            )
```
Comment (mikeldking) [inline src/phoenix/trace/exporter.py:100]: An exception seems too strong. Can we just log a warning?

Comment (RogerHYang) [reply src/phoenix/trace/exporter.py:100]: ok. will change

File: src/phoenix/server/app.py (hunk 1 above)
Comment (RogerHYang) [reply src/phoenix/server/app.py:165]: this can be a false positive if some other server is running on the same port and have the same end point

Comment (mikeldking) [reply src/phoenix/server/app.py:165]: Cool, sg

**Review by mikeldking — APPROVED** [review]

**Pull request merged**